GANDLF/data/preprocessing/rgb_conversion.py

```diff
@@ -59,9 +59,27 @@ def apply_transform(self, subject: Subject) -> Subject:
         return subject
 
 
-def rgba2rgb_transform(parameters=None):
+def rgba2rgb_transform(parameters:dict=None) -> RGBA2RGB:
```

Comment:
this and the lower one change should have type of Optional[dict] if the default is None

Reply:
Fixed.